src/firefly/js/util/WebUtil.js

```diff
@@ -351,18 +354,6 @@ export function updateSet(object, path, value) {
 }
 
 /**
- * This is a wrapper of React update's $merge for use with deep object update.
```

Comment:
yes, please. Useful! Thanks for bringing back.

On 5/9/16 1:29 PM, loitly wrote:
>
> In src/firefly/js/util/WebUtil.js 
> <https://github.com/Caltech-IPAC/firefly/pull/80#discussion_r62567320>:
>
> > @@ -351,18 +354,6 @@ export function updateSet(object, path, value) {
> >  }
> >
> >  /**
> > - * This is a wrapper of React update's $merge for use with deep object update.
>
> I removed it thinking no one was using it. Then I realized quickly 
> that Emmanuel was using it. I will add it back in.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/Caltech-IPAC/firefly/pull/80/files/8f4a66e088ebb41291604de6708df1d8dcabd878#r62567320>
>

-- 
Emmanuel Joliet
NASA/IPAC  Infrared Science Archive
MS 100-22, Caltech, Pasadena, CA 91125
Office: 170
Email:  ejoliet@ipac.caltech.edu
Phone:  (626) 395-1489